srcpkgs/texlive/template

```diff
@@ -0,0 +1,388 @@
+# Template file for 'texlive'
+pkgname=texlive
+version=20200406
+revision=1
+wrksrc="texlive-${version}-source"
+build_wrksrc="build"
+build_style=gnu-configure
+configure_script="../configure"
+configure_args="
+ --enable-biber
+ --enable-epsfwin
+ --enable-ipc
+ --enable-luajittex
+ --enable-luajithbtex
+ --enable-mfluajit
+ --enable-mfluajit-nowin
+ --enable-luatex
+ --enable-luahbtex
+ --enable-mflua
+ --enable-mfluajit
+ --enable-mfluajit-nowin
+ --enable-mftalkwin
+ --enable-regiswin
+ --enable-shared
+ --enable-tektronixwin
+ --enable-unitermwin
+ --enable-xetex
+ --enable-dvipng
+ --enable-dvipsk
+ --enable-ps2eps
+ --enable-psutils
+ --disable-bibtex-x
+ --disable-chktex
+ --disable-cjkutils
+ --disable-detex
+ --disable-dialog
+ --disable-dvi2tty
+ --disable-dvisvgm
+ --disable-largefile
+ --disable-lcdf-typetools
+ --disable-multiplatform
+ --disable-native-texlive-build
+ --disable-pdfopen
+ --disable-ps2pkm
+ --disable-t1utils
+ --disable-tex4htk
+ --disable-ttf2pk2
+ --disable-vlna
+ --disable-xindy
+ --with-ps=gs
+ --with-banner-add=/VoidLinux
+ --with-system-cairo
+ --with-system-freetype2
+ --with-system-gd
+ --with-system-graphite2
+ --with-system-gmp
+ --with-system-harfbuzz
+ --with-system-icu
+ --with-system-libpaper
+ --with-system-libpng
+ --with-system-mpfr
+ --with-system-poppler
+ --with-system-pixman
+ --with-system-teckit
+ --with-system-xpdf
+ --with-system-zlib
+ --with-system-zziplib
+ --with-xdvi-x-toolkit=Xaw"
+hostmakedepends="pkg-config perl lua52-BitOp texinfo"
+makedepends="cairo-devel freetype-devel gd-devel graphite-devel gmp-devel
+ harfbuzz-devel icu-devel libpaper-devel libpng-devel mpfr-devel
+ poppler-devel pixman-devel libteckit-devel zlib-devel zziplib-devel
+ libXaw-devel"
+depends="dialog ghostscript perl-Tk texlive-core"
```

Comment:
Should it depend on new version of triggers?

Reply:
I was discussing this with q66 and we thought no, but after doing a few more installations, it seems to be much more logical to depend on the triggers, otherwise if a user doesn't do `xbps-install -Su` before `xbps-install texlive-most` (or whatever) they'll have a broken install...

So yes I will make it depend on new triggers.